src/bp-xprofile/bp-xprofile-filters.php

```diff
@@ -111,10 +111,11 @@
 add_action( 'update_option_bp-disable-avatar-uploads', 'bp_core_xprofile_clear_all_user_progress_cache' ); // When avatar photo setting updated in wp-admin > Settings > profile.
 add_action( 'update_option_bp-disable-cover-image-uploads', 'bp_core_xprofile_clear_all_user_progress_cache' ); // When cover photo setting updated in wp-admin > Settings > profile.
 
-
-
 //Display Name setting support
 add_filter( 'bp_after_has_profile_parse_args', 'bp_xprofile_exclude_display_name_profile_fields' );
+// Repair repeater field repeated in admin side.
+add_filter( 'bp_repair_list', 'bb_xprofile_repeater_field_repair' );
+add_action( 'admin_init', 'bb_xprofile_repeater_field_repair_with_background' );
```

Comment:
@sadrul Please remove the background update here. If we have provided the repair tool that's enough.